docs/Configuration.md

```diff
@@ -140,7 +140,7 @@ Default: `"/tmp/<path>"`
 
 The directory where Jest should store its cached dependency information.
 
-Jest attempts to scan your dependency tree once (up-front) and cache it in order to ease some of the filesystem raking that needs to happen while running tests. This config option lets you customize where Jest stores that cache data on disk.
+Jest attempts to scan your dependency tree once (up-front) and cache it in order to ease some of the filesystem ranking that needs to happen while running tests. This config option lets you customize where Jest stores that cache data on disk.
```

Comment:
this is "raking" on purpose, but I don't think it's a good word. It talks about reading the same files over and over from disk. Just "access" maybe?

/cc @orta any good ideas?

Reply:
Churn?

Reply:
Yeah, I like it! English is hard 😅 

@Biki-das could you change it to that?

Reply:
yeah why not 😀, and thanks @orta for the suggestion